### DISCUSSION TITLE
uBlock Origin works against Umami

### DISCUSSION THREAD

**Discussion by zilpe** [umami-software/umami#689] [General]
I knew the 'undetectable by adblockers' too good to be true haha.

uBlock Origin blocks the endpoint, even though it's hosted on a custom domain and on a VPS.

**Comment by meirroth** [comment]
Just change the script file name ;)
Do a search and replace on everything, replacing '/umami.js' with what ever you wish.

**Reply by zilpe** [reply]
Does that really work? I would expect that to break everything haha

**Reply by meirroth** [reply]
Working perfectly fine for me :D
Just remember to do this every time there is a new update.

**Comment by flamekiller22** [comment]
@zilpe do we just have to change the js script name and that's it or is it being imported somewhere and we'd heave to change it?

**Comment by MagicLike** [comment]
What about writing an [issue](https://github.com/uBlockOrigin/uAssets/issues/new?assignees=&labels=&template=bug_report.yml)?
Eventually it could be possible, to unblock "umami.js"...

**Reply by Vultik** [reply]
@zilpe Feature added in the [v1.26.0 release](https://github.com/mikecao/umami/releases/tag/v1.26.0)
